data-pipeline/fetch_data.py

```
@@ -986,7 +1025,7 @@ def _save_to_postgres(self, data: List[Dict[str, Any]]):
 
             conn.commit()
             logger.info(
-                f"[run_id={self.run_id}] Transaction committed. Saved {inserted_count} records to PostgreSQL"
+                f"[run_id={self.run_id}] Transaction committed. Saved {attempted_count} attempted records to PostgreSQL"
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)